file.go

```diff
@@ -9,14 +9,18 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
 )
 
+const retrySleepSeconds = 1;
```

Comment:
NIT: would be a tiny bit more logical to do the `1 * time.Second` math here.

Reply:
Done.